manifests-maistra/2.0.0/maistraoperator.v2.0.0.clusterserviceversion.yaml

```diff
@@ -12,7 +12,7 @@ metadata:
       The Maistra Operator enables you to install, configure, and manage an instance of Maistra service mesh. Maistra is based on the open source Istio project.
 
     containerImage: maistra/istio-ubi8-operator:2.0.0
-    createdAt: 2020-10-12T13:32:16EDT
+    createdAt: 2020-10-20T15:49:09CEST
```

Comment:
Can this be dropped from the commit?

Reply:
I could drop it, but it's been legitimately re-generated and has new content - is the new date a problem?

Reply:
Oh, nevermind- the CRDs are separate. I'll remove the date changes